api/v1beta1/openstackdataplane_webhook.go

```diff
@@ -48,26 +44,6 @@ var _ webhook.Defaulter = &OpenStackDataPlane{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *OpenStackDataPlane) Default() {
-	openstackdataplanelog.Info("default", "name", r.Name)
-	r.Spec.Default()
-
-}
-
-// Default - set defaults for this OpenStackDataPlane spec
-func (spec *OpenStackDataPlaneSpec) Default() {
-	for nodeName, node := range spec.Nodes {
-		if node.HostName == "" {
-			node.HostName = nodeName
-		}
-		if node.Node.AnsibleSSHPrivateKeySecret == "" {
-			if node.Role == "" {
-				continue
-			}
-			roleSSHSecret := spec.Roles[node.Role].NodeTemplate.AnsibleSSHPrivateKeySecret
-			node.Node.AnsibleSSHPrivateKeySecret = roleSSHSecret
-		}
-		spec.Nodes[nodeName] = *node.DeepCopy()
-	}
 }
```

Comment:
we're not going to use the webhook to default the ssh key any longer? what is the point of the webhook at this point?